ui-components/src/helpers/tableSummaries.tsx

```diff
@@ -1,81 +1,135 @@
+      <strong>
+        {minIncomeRangeMin}
+        {t("t.perMonth")}
+      </strong>
+    )
+  } else {
+    minIncome = (
+      <>
+        <strong>{minIncomeRangeMin}</strong> {t("t.to")}{" "}
+        <strong>
+          {minIncomeRangeMax} {t("t.perMonth")}
+        </strong>
+      </>
+    )
+  }
 
-    const getRent = (rentMin: string, rentMax: string, percent = false) => {
-      const unit = percent ? `% ${t("t.income")}` : ` ${t("t.perMonth")}`
-      return rentMin == rentMax ? (
-        <>
-          <strong>{rentMin}</strong>
-          {unit}
-        </>
-      ) : (
-        <>
-          <strong>{rentMin}</strong> {t("t.to")} <strong>{rentMax}</strong>
-          {unit}
-        </>
-      )
+  const getRent = (rentMin?: string, rentMax?: string, percent = false) => {
+    const unit = percent ? `% ${t("t.income")}` : ` ${t("t.perMonth")}`
+    if (rentMin == undefined) {
+      //TODO(#345): figure out what to display when there's no data
+      return <strong>{t("t.call")}</strong>
     }
+    return rentMin == rentMax || rentMax == undefined ? (
+      <>
+        <strong>{`$${rentMin}`}</strong>
+        {unit}
+      </>
+    ) : (
+      <>
+        <strong>{`$${rentMin}`}</strong> {t("t.to")} <strong>{`$${rentMax}`}</strong>
+        {unit}
+      </>
+    )
+  }
 
-    // Use rent as percent income if available, otherwise use exact rent
-    const rent = unitSummary.rentAsPercentIncomeRange.min
-      ? getRent(
-          unitSummary.rentAsPercentIncomeRange.min.toString(),
-          unitSummary.rentAsPercentIncomeRange.max.toString(),
-          true
-        )
-      : getRent(unitSummary.rentRange.min, unitSummary.rentRange.max)
+  // Use rent as percent income if available, otherwise use exact rent
+  const rent = rentAsPercentIncomeRangeMin
+    ? getRent(rentAsPercentIncomeRangeMin, rentAsPercentIncomeRangeMax, true)
+    : getRent(rentRangeMin, rentRangeMax)
 
-    return {
-      unitType: <strong>{t(`listings.unitTypes.${unitSummary.unitType.name}`)}</strong>,
-      minimumIncome: (
-        <>
-          {minIncome} {t("t.perMonth")}
-        </>
-      ),
-      rent: <>{rent}</>,
-      availability: (
-        <>
-          {unitSummary.totalAvailable > 0 ? (
-            <>
-              <strong>{unitSummary.totalAvailable}</strong>{" "}
-              {unitSummary.totalAvailable == 1 ? t("t.unit") : t("t.units")}
-            </>
-          ) : (
-            <span className="uppercase">{t("listings.waitlist.label")}</span>
-          )}
-        </>
-      ),
-      totalCount: (
-        <>
-          <strong>{unitSummary.totalCount}</strong>{" "}
-          {unitSummary.totalCount == 1 ? t("t.unit") : t("t.units")}
-        </>
-      ),
-    }
-  })
+  return {
+    unitType: <strong>{t(`listings.unitTypes.${unitTypeName}`)}</strong>,
+    minimumIncome: <>{minIncome}</>,
+    rent: <>{rent}</>,
+    availability: (
+      <>
+        {totalAvailable && totalAvailable > 0 ? (
```

Comment:
nit: I think this can just be `{totalAvailable > 0 ? (`

Reply:
This was here in case totalAvailable was undefined.  I updated the function call to take care of the undefined case.